src/index.html

```diff
@@ -17,16 +17,37 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div
+      class="search-bar"
+      data-qa="big"
+    >
+      <img
+        class="image-big"
+        src="images/Search.svg"
+        alt="search-big"
+      />
```

Comment:
The checklist specifies that the icon should be implemented using the background-image CSS property instead of an 'img' tag.